capstone/src/test/java/com/google/step/search/ElasticsearchClientTest.java

```diff
@@ -71,4 +88,96 @@ public LowLevelHttpResponse execute() {
     testClient.updateRestaurantHeader(HEADER_1);
   }
 
+  @Test
+  public void testQueryRestaurants() throws IOException {
+    HttpTransport transport = new MockHttpTransport() {
+      @Override
+      public LowLevelHttpRequest buildRequest(String method, String url) throws IOException {
+        assertTrue(method.equalsIgnoreCase("POST"));
+
+        URL urlFromString = new URL(url);
+        assertEquals(elasticsearchHostname, urlFromString.getHost());
+        assertEquals(elasticsearchPort, urlFromString.getPort());
+        assertEquals("/restaurants/_search", urlFromString.getPath());
+
+        return new MockLowLevelHttpRequest() {
+          @Override
+          public LowLevelHttpResponse execute() throws IOException {
+
+            String expectedQueryBody = new JSONObject()
+                .put("query", new JSONObject()
+                    .put("multi_match", new JSONObject()
+                        .put("query", "goog")
+                        .put("fields", new JSONArray(Arrays.asList("name", "cuisine")))))
+                .toString();
+
+            String queryRequestBody = getContentAsString();
+
+            assertEquals(expectedQueryBody, queryRequestBody);
+
+            String stringContent = new JSONObject()
+                .put("hits", new JSONObject()
+                    .put("hits", new JSONArray()
```

Comment:
Double-hits?

Reply:
Perhaps I should leave a comment in about this, but that's how the Elasticsearch response is formatted.